headingoffset-polyfill.test.html

```diff
@@ -40,6 +40,14 @@
           it("default value when unset", () => {
             expect(container.headingOffset).to.equal(0);
           });
+          it("default value when invalid", () => {
+            container.headingOffset = "fish";
+            expect(container.headingOffset).to.equal(0);
+          });
+          it("allows negative values", () => {
+            container.headingOffset = -1;
+            expect(container.headingOffset).to.equal(-1);
```

Comment:
I thought it does not allow negative values, and this should also return 0?

Reply:
You are correct! Here are the relevant lines from the implementation of `GetHeadingOffset` in [ax_node_object.cc (line 2968–2994)](https://chromium-review.googlesource.com/c/chromium/src/+/5445406/13/third_party/blink/renderer/modules/accessibility/ax_node_object.cc):

```C
if (ancestorHeadingOffset > 0) {
  offset += ancestorHeadingOffset;
}
```

In other words, if `ancestorHeadingOffset` is negative, it’s _not_ added into `offset`.

Reply:
(Converting this PR to draft while I fix that.)

Reply:
Fixed via https://github.com/smockle/headingoffset-polyfill/pull/14/commits/36d82071c9dc6c363adcad4a21c185462a2a80c4!